lib/jest-helpers.js

```diff
@@ -142,6 +142,7 @@ async function getExamples (componentName) {
 function renderSass (options) {
   return sassRender({
     includePaths: sassPaths,
+    quietDeps: true,
```

Comment:
Whilst this PR fixes `govuk-frontend` deprecation warnings we add `quietDeps: true` to ignore:

```console
govuk_frontend_toolkit
govuk-elements-sass
```

See https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#silence-deprecation-warnings-from-dependencies-in-dart-sass